### ISSUE TITLE
Redesign vote status & refine LP mechanic to update market

### ISSUE THREAD

**Issue by ckpringle** [vegaprotocol/frontend-monorepo#3601]
Note - vote status also appears here: https://explorer.vega.xyz/governance

## Story 
As a user 
I want to immediately understand the status of the vote for or against a proposal and how I can cast my own vote
So that I understand what's going to happen to this proposal, and can more obviously see the impact of governance at vega in action

### All Votes excl. Market Updates

- [x] I can see two vote status bars, one showing majority voting yes/no, and the other showing participation progress. The maximum value for the progress bar should be the threshold for that proposal type. 
- [x] When majority is reached (threshold is a network param) the majority indicator text that sits on top of the majority progress bar says above it turns green and states "[network param]% majority threshold met". Otherwise it says "[network param]% majority threshold not met".
- [x] When the participation bar reaches 100% (i.e. the network param threshold is met) the indicator text on top of it reads "[network param]% participation threshold met". Otherwise it says "[network param]% participation threshold not met". 
- [x] If the participation exceeds the required amount continue to show the full amount on the participation status bar, and the bar should continue to just be full
- [x] There should be two text areas beneath the majority status bar. On the left hand side: "Token votes for", with the number expressed in millions with 1 DP, and as a rounded percentage. These values should include a tooltip with more detailed figures. On the right "Token votes against", with the same requirements.
- [x] Under the participation progress bar, there should be text saying "Total tokens voted", with the same value, percentage and tooltip requirements as the text for the majority bar.
- [x] If the proposal is open, I can still see whether the proposal is currently set to pass / fail 
- [x] I can see how long is left to vote

For the proposal details page:

- [x] The "Vote Breakdown" section has been removed as this is now all covered elsewhere
- [x] The "Cast Vote" section is now shown after the description and market / asset details sections
- [x] The "Full proposal JSON" section is now shown at the end after "Cast Vote"

### Market Update votes

- [x] When the vote is a market update I can see an additional set of 2 graphs that capture the LP vote, with the information sourced from the relevant place on the API.
- [x] When both liquidity and token votes data is shown, the two areas should have titles to help the user differentiate. 
- [x] Text and rendering logic follows the same patterns as the token vote requirements above, with the following differences:
- [x] "Token votes for/against" becomes "LP votes for/against"
- [x] "Total tokens voted" becomes "Total LP tokens voted"

## Tasks 

- [x] Explore and sketch
- [x] Design - and tighten the design of the page as a whole so this fits in context 
- [x] Team and stakeholder review

## Additional details / background info

Past sketches

![image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/6e240877-4155-4fe6-8d40-f2f168688656)
![image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/79d9e735-2741-4162-99bf-454e3f3026ff)
![image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/f77d2dc2-8571-44ae-86e3-2d4693dbebbb)
![image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/e57d28d8-3401-443c-b684-df28fbbc88ab)
![image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/dce0a199-2b84-4c1c-864f-c57a48a1def0)

<img width="919" alt="Screenshot 2023-05-04 at 11 18 13" src="https://user-images.githubusercontent.com/46783148/236177053-16b3440c-b724-4b34-8878-d5f2b60c3c47.png">
<img width="1231" alt="Screenshot 2023-05-04 at 11 18 03" src="https://user-images.githubusercontent.com/46783148/236177056-a26a9631-87de-48d4-aec6-c347b9b98ce3.png">

Snapshot have an established pattern https://snapshot.org/#/aave.eth/proposal/0xb477feee6d506be940154bc84278654f2863044854f80fca825b236253a97778

Generally ours is difficult to understand - participants (who work on the project) were left unsure what was being said in the participation section. 

<img width="1136" alt="Screenshot 2023-03-06 at 13 28 53" src="https://user-images.githubusercontent.com/13255539/223123521-b2c054fa-989a-4a59-8eba-520ca57bd2e5.png">

**Comment by JonRay15** [comment]
Updated to include Barnet;s feedback, namely 

1 Wanting something to indicate the association between  the bars and the labels at the top ... so I've added a little box to group them so its more obvious
2 Wanting to have the red in the no votes ... which I'm personally not sold on but for now I'm including

I've also tried sort of fading out the bar that has been overrulled in the case of the market update.  His preference was for some way of de-emphasising that but how we didnt really decide. Fading maybe is better than turning it grey?  Dont know.
